docs/src/app/components/pages/components/DropDownMenu/ExampleLongMenu.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import DropDownMenu from 'material-ui/lib/DropDownMenu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+const longMenuItems = [];
+for (let i = 0; i < 100; i++ ) {
+  longMenuItems.push({
+    value: i,
+    text: `Item ${i}`,
+  });
+}
+
+export default class DropDownMenuLongMenuExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {value: 10};
+  }
+
+  handleChange = (e, index, value) => this.setState({value});
+
+  render() {
+
+    const items = longMenuItems.map((item, index) => (
```

Comment:
Well, my point is, we can save 100 class allocation each time the value change.
But each, it may be insignificant.